CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.3.0
+
+`2021-05-XX`
+
+### What's new
+
+- **Added bottom reached and row intersection callbacks to the `Table`.**
```

Comment:
Could reword and explicitly mention lazy loading.

```suggestion
- **Added lazy loading to `Table`** with new `onBottomReached` and `onRowIntersection` props.
```

Reply:
Changed.